src/backtrace.cc

```diff
@@ -0,0 +1,5 @@
+#include "src/base/logging.cc"
```

Comment:
`#include "src/base/logging.h"` works too, at least locally for me. Any particular reason for including the source file itself?